### PR TITLE
Add a few tooltips

### PR DESCRIPTION
**Pull request by G-Ray**
### Description
- Add a few tooltips on actions across the app
- Add `mapbox/locale.js` to patch mapbox strings with our i18n process

I got an overwhelming diff when generating i18n files, so I did not add new string to po files. Should I do something ?

I know scoping has no effect on our i18 strings for now, but I let some "generic" reusable strings like close or menu without scoping.

### CODE REVIEW

File: src/mapbox/extended_nav_control.js [hunk 1]
```diff
@@ -10,11 +10,19 @@ export default class ExtendedControl {
     this.bottomButtonGroup = document.createElement('div');
 
     const buttonClass = 'icon-plus map_control_group__button__zoom';
-    this._zoomInButton = this._createButton(buttonClass, 'Zoom In', () => this._map.zoomIn());
+
+    this._zoomInButton = this._createButton(
+      buttonClass,
+      'Zoom In',
+      () => this._map.zoomIn(),
+      'Zoom +'
```
Comment (bbecquet) [inline src/mapbox/extended_nav_control.js:18]: There is no added value here to use a different text in `title` and `aria-label`, you can just re-use the same string for both.

File: src/adapters/scene.js [hunk 2]
```diff
@@ -68,6 +69,7 @@ Scene.prototype.initMapBox = function() {
     center: this.center,
     hash: false,
     maxZoom: 20,
+    locale,
```
Comment (bbecquet) [inline src/adapters/scene.js:72]: :+1:

File: src/panel/direction/VehicleSelector.jsx [hunk 3]
```diff
@@ -15,6 +23,7 @@ const VehicleSelector = ({ vehicles, activeVehicle, onSelectVehicle }) =>
       )}
       onClick={() => onSelectVehicle(vehicle)}
       aria-label={vehicle}
```
Comment (bbecquet) [inline src/panel/direction/VehicleSelector.jsx:25]: You can apply the same localization to aria-label

File: src/mapbox/extended_nav_control.js (hunk 1 above)
Comment (G-Ray) [reply src/mapbox/extended_nav_control.js]: Ok, title is set according to ariaLabel now

**Pull request merged**